src/ElmSharp/ElmSharp/PreloadedWindow.cs

```diff
@@ -14,13 +14,21 @@
  * limitations under the License.
  */
 
+using System;
+using System.ComponentModel;
+using System.Reflection;
+
 namespace ElmSharp
 {
-    internal class PreloadedWindow : Window
+    /// <summary>
+    /// Pre-created window which prepares features that takes time in advance.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
I changed `PreloadedWindow` class to public so that `ElmSharp.Wearable.Preload` can access the properties of it.
Internal visible to `ElmSharp.Wearable` dll could be the option, but that brings the ambiguous of `Interop` classes between `ElmSharp` and `ElmSharp.Wearable`.